jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,356 @@
+    
+    curl --header "Circle-Token: $CIRCLECI_TOKEN" \
+         --header 'Accept: application/json'    \
+         --header 'Content-Type: application/json' \
+         https://circleci.com/api/v2/project/gh/{USER_NAME}/hello-world/pipeline 
+    ```
+
+    You will likely receive a long string of unformatted JSON. After formatting, it should look like so:
+
+    ```sh
+    {
+      "next_page_token": null,
+      "items": [
+        {
+          "id": "03fcbba0-d847-4c8b-a553-6fdd7854b893",
+          "errors": [],
+          "project_slug": "gh/{YOUR_USER_NAME}/hello-world",
+          "updated_at": "2020-01-10T19:45:58.517Z",
+          "number": 1,
+          "state": "created",
+          "created_at": "2020-01-10T19:45:58.517Z",
+          "trigger": {
+            "received_at": "2020-01-10T19:45:58.489Z",
+            "type": "api",
+            "actor": {
+              "login": "teesloane",
+              "avatar_url": "https://avatars0.githubusercontent.com/u/12987958?v=4"
+            }
+          },
+          "vcs": {
+            "origin_repository_url": "https://github.com/{YOUR_USER_NAME}/hello-world",
+            "target_repository_url": "https://github.com/{YOUR_USER_NAME}/hello-world",
+            "revision": "ca67134f650e362133e51a9ffdb8e5ddc7fa53a5",
+            "provider_name": "GitHub",
+            "branch": "master"
+          }
+        }
+      ]
+    }
+    ```
+
+    
+    That's great! Hopefully everything is working for you up to this point. Let's move on to performing something that might be a bit more useful.
+
+5. One of the benefits of the CircleCI API v2 is the ability to remotely trigger pipelines with parameters. The following code snippet simply triggers a pipeline via `curl` without any body parameters:
+
+    ```sh
+    curl -X POST https://circleci.com/api/v2/project/gh/{YOUR_USER_NAME}/hello-world/pipeline \
+      -H 'Content-Type: application/json' \
+      -H 'Accept: application/json' \
+      -H "Circle-Token: $CIRCLECI_TOKEN" \
+      
+    # Which returns:
+    {
+      "number": 2,
+      "state": "pending",
+      "id": "e411ea74-c64a-4d60-9292-115e782802ed",
+      "created_at": "2020-01-15T15:32:36.605Z"
+    }
+    ```
+
+    While this alone can be useful, we want to be able to customize parameters of the pipeline when we send this `POST` request. By including a body parameter in the `curl` request (via the `-d` flag), we can customize specific attributes of the pipeline when it runs: pipeline parameters, the branch, or the git tag. Below, we are telling the pipelines to trigger for "my-branch"
+
+    ```sh
+    curl -X POST https://circleci.com/api/v2/project/gh/{YOUR_USER_NAME}/hello-world/pipeline \
+      -H 'Content-Type: application/json' \
+      -H 'Accept: application/json' \
+      -H "Circle-Token: $CIRCLECI_TOKEN" \
+      -H 'x-attribution-login: string' \
+      -H 'x-attribution-actor-id: string' 
+      -d "branch=my-branch"
+    ```
```

Comment:
<img width="1161" alt="Screenshot 2020-01-20 at 11 56 39" src="https://user-images.githubusercontent.com/24589403/72724711-025f3980-3b7c-11ea-8572-6f7f7d490fed.png">

when I try this the pipeline is triggered and then the `-d "branch=my-branch"` part is there ready to be run on it's own, rather than being part of the API post thingy - see screenshot - hope this makes sense!